docs/guides/soloproject/topics/tier_developer_python.md

```diff
@@ -0,0 +1,95 @@
+# Developer Tiers ( Non Web Developer - Python )
+![Solo Project topic image](../assets/horizontal-paint-splash-yellow.jpg)
+<table>
+<tr>
+<td><strong>Tier</strong></td>
+<td><strong>Requirements</strong></td>
+</tr>
+<tr>
+<td>Tier&nbsp;1: Foundations (Getting Started)</td>
+<td>
+<p>This tier is for developers who are just starting out in development in python.</p>
+<strong>Skills:</strong>
+<ul>
+    <li><strong>Basic Syntax & Structure:</strong> Variables, data types, operators, and indentation.</li>
+    <li><strong>Control Flow:</strong> if, else, elif, for, while, and simple boolean logic.</li>
+    <li><strong>Functions:</strong> Defining and calling functions, parameters, and return values.</li>
+    <li><strong>Data Structures:</strong> Lists, dictionaries, tuples, and sets basics.</li>
+    <li><strong>Basic File I/O:</strong> Reading from and writing to files using open().</li>
+    <li><strong>Comments & Documentation:</strong> Writing inline comments and basic docstrings.</li>
+</ul>
+<strong>Tools</strong>
+<ul>
+    <li>Code Editor: Familiarity with a code editor such as Visual Studio Code, Neovim, Webstorm, etc.</li>
+    <li>Git/GitHub: Basic understanding of how to use Git/GitHub. Familiarity with cloning repositories, committing changes, and pushing code.</li>
+    <li>Deployment Tools: Hosting services to deploy static web pages such as GitHub Pages, Netlify, Vercel, Render, Railway, etc.</li>
+</ul>
+
+<p>Your Solo Project <strong><em>must</em></strong> showcase at least 4/6 of the above skills. The project must be deployed, or a demo video can be used instead of a deployment.</p>
+
+<p>The solo project must include a readme, less than 12 month old, and must be your own work (not following tutorials).</p> 
```

Comment:
`solo project` should be `Solo Project`.

Also, suggest replacing `not following tutorials` with `not following tutorials and not using AI to generate your code`.